oaknational/Oak-Web-Application
# PUPIL-457 Fix/start lesson label bug

## Description

Music year: 1951

overview next section label wasnt being driven by lesson state.lessonStarted

now using lesson sate to dictate label

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2289--oak-web-application.netlify.thenational.academy/pupils/l/shakespearean-comedy-the-tempest-88f0/lessons/context-of-the-tempest-cctp4c
2. Click through lesson
3. You should see once intro is complete the label should say continue lesson untilo the lesson is finished


## Checklist
